[ERROR] could not convert string to float: "119.70731707317071368')"

Found this in the logs in OWS 1.9.9:
```
[2026-02-20 10:20:08,295] [ERROR] could not convert string to float: "119.70731707317071368')"
Traceback (most recent call last):
  File "/app/lib/python3.12/site-packages/datacube_ows/ogc.py", line 125, in ogc_svc_impl
    return version_support.router(nocase_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/lib/python3.12/site-packages/datacube_ows/utils.py", line 38, in log_wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/app/lib/python3.12/site-packages/datacube_ows/wms.py", line 29, in handle_wms
    return get_map(nocase_args)
           ^^^^^^^^^^^^^^^^^^^^
  File "/app/lib/python3.12/site-packages/datacube_ows/utils.py", line 38, in log_wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/app/lib/python3.12/site-packages/datacube_ows/data.py", line 81, in get_map
    params = GetMapParameters(args)
             ^^^^^^^^^^^^^^^^^^^^^^
  File "/app/lib/python3.12/site-packages/datacube_ows/wms_utils.py", line 346, in __init__
    self.geometry = _get_polygon(args, self.crs)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/lib/python3.12/site-packages/datacube_ows/wms_utils.py", line 95, in _get_polygon
    minx, miny, maxx, maxy = _get_geobox_xy(args, crs)
                             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/lib/python3.12/site-packages/datacube_ows/wms_utils.py", line 68, in _get_geobox_xy
    miny, minx, maxy, maxx = map(float, args["bbox"].split(","))
    ^^^^^^^^^^^^^^^^^^^^^^
ValueError: could not convert string to float: "119.70731707317071368')"
```
I don't know what the initial request was, but the `')` at the end of the number looks like some quoting+ending parenthesis.

I'm guessing some part of stripping formatting from input failed to do its job, and then caused the problem triggering the backtrace. Converting a string to a float can fail for a bunch of other reasons as well, so I will make a PR that catches that problem so the backtrace isn't triggered.

The input validation/tidying still needs to be fixed and I don't know how to do that, so leaving that work for this issue.

--- a/datacube_ows/wms_utils.py
+++ b/datacube_ows/wms_utils.py
@@ -64,10 +64,15 @@ def _bounding_pts(
 
 
 def _get_geobox_xy(args, crs: CRS) -> tuple[float, float, float, float]:
-    if get_config().published_CRSs[str(crs)]["vertical_coord_first"]:
-        miny, minx, maxy, maxx = map(float, args["bbox"].split(","))
-    else:
-        minx, miny, maxx, maxy = map(float, args["bbox"].split(","))
+    try:
+        if get_config().published_CRSs[str(crs)]["vertical_coord_first"]:
+            miny, minx, maxy, maxx = map(float, args["bbox"].split(","))
+        else:
+            minx, miny, maxx, maxy = map(float, args["bbox"].split(","))
+    except ValueError:
+        raise WMSException(
+            "Invalid bbox parameter.  Must be 4 decimal numbers separated by commas."
+        ) from None
     return minx, miny, maxx, maxy
 
 

--- a/tests/test_wms_utils.py
+++ b/tests/test_wms_utils.py
@@ -344,6 +344,41 @@ def test_get_geobox() -> None:
     OWSConfig._instance = None
 
 
+def test_invalid_bbox() -> None:
+    from datacube_ows.ows_configuration import OWSConfig
+
+    mock_cfg = MagicMock()
+    OWSConfig._instance = mock_cfg
+    with pytest.raises(WMSException):
+        _ = datacube_ows.wms_utils._get_geobox(
+            args={
+                "width": "256",
+                "height": "256",
+                "bbox": "Not valid - not even close",
+            },
+            crs=CRS("EPSG:4326"),
+        )
+    with pytest.raises(WMSException):
+        _ = datacube_ows.wms_utils._get_geobox(
+            args={
+                "width": "256",
+                "height": "256",
+                "bbox": "-43.28507087113431,sneaky sneaky,-43.28507087113431,146.64289867785524",
+            },
+            crs=CRS("EPSG:4326"),
+        )
+    with pytest.raises(WMSException):
+        _ = datacube_ows.wms_utils._get_geobox(
+            args={
+                "width": "256",
+                "height": "256",
+                "bbox": "-43.28507087113431,145.77444644,-43.28507087113431,146.64289867785524')",
+            },
+            crs=CRS("EPSG:4326"),
+        )
+    OWSConfig._instance = None
+
+
 def test_get_arg() -> None:
     val = datacube_ows.wms_utils.get_arg({"myval": "Oh Yeah"}, "myval", "My Value")
     assert val == "Oh Yeah"

